cudax/test/event/event_smoke.cu

```diff
@@ -84,33 +86,50 @@ TEST_CASE("can use event_ref to record and wait on an event", "[event]")
 
 TEST_CASE("can construct an event with a stream_ref", "[event]")
 {
-  test::stream stream;
-  cudax::event ev(stream.ref());
+  cudax::stream stream;
+  cudax::event ev(static_cast<cuda::stream_ref>(stream));
```

Comment:
Shouldnt this be implicitly convertible?

Reply:
It should, I think we test that in other place and this test case was designed to verify you can pass in the stream_ref